src/aleph/http/multipart.clj

```diff
@@ -207,15 +260,48 @@
    flag to know whether content might be read directly from `:content` or
    should be fetched from the file specified in `:file`.
 
+   Each part should be released using `netty/release` helper to cleanup
+   allocated buffers and temp files (if any) as soon as the data is fully
+   consumed (i.e. temp file moved to a target location). Note, it's also
+   safer to close the stream of chunks manually to ensure all chunks that
+   were never read from the stream but were already consumed from the connection,
+   are also deallocated.
+
+   Typical usage looks like:
+
+   ```
+   (require '[aleph.http.multipart :as multipart])
+   (require '[aleph.netty :as netty])
+   (require '[manifold.stream :as stream])
+   (require '[clojure.java.io :as io])
+
+   (defn file-upload-handler [req]
+     (let [chunks (multipart/decode-request req)]
+       (d/chain'
+         (stream/take! chunks)
+         (fn [{:keys [file] :as chunk}]
```

Comment:
These are _parts_, not _chunks_, correct? We should be precise. IIUC, chunks may be incomplete pieces of a part, and a part corresponds to everything between the multipart boundaries (minus headers, etc), e.g., a whole file. If that's right, we should indicate that `decode-request` returns a stream of parts, not chunks.

Reply:
Agreed.

Reply:
Actually, it has been called `chunk` because we introduced the `MultipartChunk` record...
Renaming `chunk` to `part` makes sense on one hand but not on the other.

Reply:
Ehh, this is confusing. How can we bring clarity?